Source/WebKit/WebProcess/WebPage/Cocoa/UnifiedTextReplacementController.mm

```diff
@@ -556,6 +624,25 @@
     return WebCore::makeSimpleRange(m_contextRanges.get(uuid));
 }
 
+std::optional<WebCore::SimpleRange> UnifiedTextReplacementController::contextRangeForSessionOrRangeWithUUID(const WTF::UUID& uuid) const
```

Comment:
I think this function should just be `contextRangeForRangeWithUUID` instead of doing two different things (and then move the session range check to the call site), especially since it's only called from one place.

Reply:
This will be called from more places in the future.